pagopa/pn-frontend
# PN-11331 - Remove TabIndex from headings

## Short description
Remove useless tabIndex 

## List of changes proposed in this pull request
- Remove tabIndex attribute

## How to test
Start login and navigate the page with tab key, you should be able to focus only interactive elements
